packages/components/tabs/src/tabs.tsx

```diff
@@ -212,15 +219,19 @@ const Tabs = defineComponent({
             ns.is(props.tabPosition),
           ]}
         >
-          <TabNav
-            ref={nav$}
-            currentName={currentName.value}
-            editable={props.editable}
-            type={props.type}
-            panes={panes.value}
-            stretch={props.stretch}
-            onTabClick={handleTabClick}
-            onTabRemove={handleTabRemove}
+          <TabNavRenderer
```

Comment:
What is the purpose of this modification?

Reply:
It's to execute the creation of  `TabNav` VNode after `panels` were patched. Without it, the `panes.value` could be stale when creating `TabNav` VNode.

Reply:
Thanks for the explanation.